tests/test_generic/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from localflavor.generic.models import IBANField
+
+
+class Partner(models.Model):
```

Comment:
Can you rename this to something like `UseNordeaExtentions` to match `IncludedCountriesModel`? Same applies to the form.